redash/query_runner/query_results.py

```diff
@@ -134,6 +137,7 @@ def create_table(connection, table_name, query_results):
         column_list=column_list,
         place_holders=",".join(["?"] * len(columns)),
     )
+    logger.debug("INSERT template: %s", insert_template)
```

Comment:
Pretty sure we don't want debugging statements being unconditionally run.  At least with the change above it, it seems to be done conditionally there.

Reply:
@justinclift thank you for the reply! So there are 2 things:
- `logger` only logs (prints to stdout/stderr) messages, nothing is actually run/executed. The default log level is [INFO](https://github.com/getredash/redash/blob/372adfed6bab0a6353a2d6801ace6044e25d29cb/redash/settings/__init__.py#L212), so `logger.debug` is actually a condition which logs the data only at a more verbose level
- for [another](https://github.com/getredash/redash/pull/6905/files#diff-b4732899338876f6f626673bd3c097e055c3cb8e8d80e016e3d24e6f8759e912R115) added logger there is a condition that checks the current log level and prevents going through too many checks unless its DEBUG (which is not default)

Reply:
ping @justinclift

Reply:
@vtatarin Sorry, but I'm short on time for the next few weeks.

Am getting some (Redash related) stuff deployed to a data centre, and that's taking the majority of my focus time.  When that's done I'll be able to look at PRs properly. :smile: